src/send/SelectRecipientButtons.tsx

```diff
@@ -149,7 +150,7 @@ export default function SelectRecipientButtons({ onContactsPermissionGranted }:
   }
 
   const onPressJumpstart = () => {
-    AppAnalytics.track(JumpstartEvents.send_select_recipient_jumpstart)
+    // AppAnalytics.track(JumpstartEvents.send_select_recipient_jumpstart)
```

Comment:
Just commented to force tests to fail so I don't forget to re-work this if needed